class-PBS-Media-Manager-API-Client.php

```diff
@@ -695,20 +695,24 @@ public static function alter_pbs_image($image_url, $operation, $width, $height,
    *
    * @param string $tp_media_id
    *   TP Media Object ID.
+   * 
+   * @param array $queryargs
+   *   The arguments for the query.
    *
    * @return array|mixed
    *   Returns an asset.
    */
-  public function get_asset_by_tp_media_id($tp_media_id) {
+  public function get_asset_by_tp_media_id($tp_media_id, $queryargs=array()) {
     /* Returns the corresponding asset if it exists.  Note that they're
      * calling it tp_media_id, NOT tp_media_object_id */
-    $query = "/assets/legacy/?tp_media_id=" . $tp_media_id;
-    $response = $this->get_request($query);
+    $endpoint = "/assets/legacy/?tp_media_id=" . $tp_media_id;
+    $response = $this->get_request($endpoint);
```

Comment:
I may be missing it, but is there a reason to perform this query without the `$queryargs` if they are passed in?